### PR TITLE
Variables cleanup & README colors

### PR DESCRIPTION
**Pull request by paulfantom**
- a little bit cleaner readme (now with colors!)
- removed variables which are defined in `vars` but used only once or never to increase readability
- rename (now empty) files with variables related to specific OS family. Use lowercase naming.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 - name: include os-specific variables
-  include_vars: "{{ ansible_os_family }}.yml"
+  include_vars: "{{ ansible_os_family | lower }}.yml"
```
Comment (atosatto) [inline tasks/main.yml]: I think we can just remove this task considering that now we don't depend anymore on any OS specific variable.

File: README.md [hunk 2]
```diff
@@ -1,75 +1,95 @@
...
-    minio_server_addr: ":9091"
+```yaml
+minio_server_addr: ":9091"
+```
 
 The Minio server listen address.
 
-    minio_server_datadirs: [ ]
+```yaml
+minio_server_datadirs: [ ]
```
Comment (atosatto) [inline README.md]: Shall we add also the `minio_server_make_datadirs` variable?

Comment (paulfantom) [reply README.md]: done

**Comment by SuperQ** [comment]
Oh, I forgot, can you update the example playbook to match the [new style](https://github.com/atosatto/ansible-minio/blob/master/molecule/default/playbook.yml).

**Pull request merged**